### PR TITLE
switch registration to djoser

### PR DESCRIPTION
**Pull request by Murrengan**

### CODE REVIEW

File: src/views/murren/SignUp.vue [hunk 1]
```diff
@@ -154,12 +165,15 @@
...
           this.goHome()
+          this.showCheckEmail()
           return
         }
 
         this.uniqueName = result.uniqueName
         this.uniqueEmail = result.uniqueEmail
         this.passwordIsTooCommon = result.passwordIsTooCommon
+        this.passwordIsTooSimilarToUsername = result.passwordIsTooSimilarToUsername
+        this.passwordIsTooSimilarToEmail = result.passwordIsTooSimilarToEmail
```
Comment (ydreiz) [inline src/views/murren/SignUp.vue:176]: It is possible to implement such validation on the frontend with the vuelidate package to ease the backend load.

File: src/store/auth.js [hunk 2]
```diff
@@ -73,50 +84,46 @@ export default {
...
-        if (data.email_sent_successfully) {
-          results.emailIsSent = true
-        }
+        const {data} = await axios.post('/auth/users/reset_password/', payload)
 
-        if (data.error_on_backend && data.error_text === 'Murren matching query does not exist.') {
-          results.notFoundMurren = true
+        if (data.status === 204) {
+          results.emailIsSent = true
         }
```
Comment (ydreiz) [inline src/store/auth.js]: Destructuring is applied here, therefore in the data variable there is only a backend data, if you need to get status you can go in several ways

```
const {data, status} = await axios.post()
status === 204

const {data, ...response} = await axios.post()
response.status === 204

const response = await axios.post()
response.status === 204
```

File: src/store/auth.js [hunk 3]
```diff
@@ -28,43 +28,54 @@ export default {
...
-          uniqueEmail: false,
-          passwordIsTooCommon: false,
-          recaptchaError: false,
-        }
+        const {data} = await axios.post('/auth/users/', payload)
 
-        const {data} = await axios.post('/murren/register/', payload)
+        if (data.email === payload.email) {
+          results.murrenIsCreated = true
+        }
```
Comment (ydreiz) [inline src/store/auth.js]: Here you can check the status of the response
201 Created

```
const {status} = await axios.post('/auth/users/', payload)

if (status === 201) {
  results.murrenIsCreated = true
}
```

**Review by ydreiz — CHANGES_REQUESTED** [review]
A couple of places where you need to fix, did not notice anything else.

File: src/store/auth.js (hunk 2 above)
Comment (Murrengan) [reply src/store/auth.js]: done

File: src/store/auth.js (hunk 3 above)
Comment (Murrengan) [reply src/store/auth.js]: Благодарю за ревью! Поправил 2 из 3 и завел таску на vuelidate

**Pull request merged**